### PR TITLE
fix some errors in  bidi algorithm when displaying arabic

### PR DESCRIPTION
**Pull request by Bluelongshirt**
fix the bug [https://github.com/lvgl/lvgl/issues/2712](url).

### CODE REVIEW

File: src/misc/lv_bidi.c [hunk 1]
```diff
@@ -465,6 +469,16 @@ static lv_base_dir_t get_next_run(const char * txt, lv_base_dir_t base_dir, uint
         pos_conv_i++;
         next_dir  = lv_bidi_get_letter_dir(letter);
         if(next_dir == LV_BASE_DIR_NEUTRAL)  next_dir = bracket_process(txt, i, max_len, letter, base_dir);
+   
+
```
Comment (kisvegabor) [inline src/misc/lv_bidi.c]: Please remove these extra empty lines.

File: src/misc/lv_bidi.c [hunk 2]
```diff
@@ -440,11 +440,15 @@ static lv_base_dir_t get_next_run(const char * txt, lv_base_dir_t base_dir, uint
     /*Find the first strong char. Skip the neutrals*/
     while(dir == LV_BASE_DIR_NEUTRAL || dir == LV_BASE_DIR_WEAK) {
         letter = _lv_txt_encoded_next(txt, &i);
+        
         pos_conv_i++;
         dir = lv_bidi_get_letter_dir(letter);
         if(dir == LV_BASE_DIR_NEUTRAL)  dir = bracket_process(txt, i, max_len, letter, base_dir);
+ 
+         if(dir==LV_BASE_DIR_LTR || dir==LV_BASE_DIR_RTL)
```
Comment (kisvegabor) [inline src/misc/lv_bidi.c]: Please remove the extra spaces from here.

**Review by kisvegabor — CHANGES_REQUESTED** [review]
Thank you! Looks good in general, but please fix some formatting issues.

**Comment by Bluelongshirt** [comment]
I am sorry that there is a problem with the format of the file, I fixed the problem, you see if it is correct

**Comment by kisvegabor** [comment]
Looks good, thank you!

**Pull request merged**